keyboards/3.0/mt.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard SYSTEM "../dtd/ldmlKeyboard.dtd">
+<keyboard locale="mt" conformsTo="techpreview">
+    <locales>
+        <!-- English is also an official language in Malta.-->
+        <locale id="en" />
+    </locales>
+    <info author="Steven R. Loomis" normalization="NFC" layout="QWERTY" indicator="MT" />
```

Comment:
Authorship I hope will also be trackable by git commit / github PR. Because it's important to loop people in when we change things!

Reply:
definitely.  may want to add more metadata.  I should have put my id in with an email.